src/containerapp/azext_containerapp/custom.py

```diff
@@ -1958,8 +1958,8 @@ def create_or_update_github_action(cmd,
                     return r
 
             raise ValidationError(
-                "Failed to find workflow file for Container App '{}' in .github/workflow folder for repo '{}'. ".format(name, repo) +
-                "If this file was removed, please use the 'az containerapp github-action delete' command to disconnect the removed workflow file connection.")
+                "Exhausted the number of re-tries allotted to polling the creation of the workflow file for Container App '{}' in .github/workflow folder for repo '{}'. ".format(name, repo) +
```

Comment:
This change was made to make sure we accurately let the customer know that the operation to get the workflow file created did not success after 6 re-tries but it's likely that the customer would see it in their repository after the create operation finishes.